### PR TITLE
Removed issues guide

### PR DESCRIPTION
**Pull request by RichardLitt**
The issues guide is badly outdated. I pulled out the only bit that still
seems relevant to our workflow - the labels - and added it into contributing.
The rest is mostly about Waffle, which we don't use anymore. This should help
declutter the contributing repo.

### CODE REVIEW

File: contribution-guidelines.md [hunk 1]
```diff
@@ -141,6 +141,55 @@ Before the pull request is merged, make sure that you squash your commits into l
...
+* yellow: Documentation  
+* light red: An important topic like security 
+* orange: An important task like cleanup / refactor  
+* red: Very important stuff like bugs
+* light green: The difficulty level 
+* green: Help wanted
+* dark green: Testing  
+* blue: A topic or subsystem 
+* purple: Enhancement
+* gray: Automatic, occasionally used by external tools (like Waffle)
```
Comment (jbenet) [inline contribution-guidelines.md]: idk if it's even worth keeping-- we dont really use this

File: contribution-guidelines.md [hunk 2]
```diff
@@ -141,6 +141,55 @@ Before the pull request is merged, make sure that you squash your commits into l
...
+    * testing
+    * test_failure
+* purple
+    * enhancement
+* gray
+    * icebox
+    * backlog
+    * sprint
+    * in progress
+    * codereview
```
Comment (jbenet) [inline contribution-guidelines.md]: we've really only done this for go-ipfs and not even worked that hard at it. definitely hit the repo cost when having to create labels for every new repo

File: contribution-guidelines.md (hunk 1 above)
Comment (RichardLitt) [inline contribution-guidelines.md]: I'm 100% for throwing it out then. Shall I?

File: contribution-guidelines.md (hunk 2 above)
Comment (Dignifiedquire) [inline contribution-guidelines.md]: There is a nice cli tool that copies all labels from one repo to another, if we want to setup consistent labeling

Comment (jbenet) [inline contribution-guidelines.md]: good to know-- do we want to? 

cc @whyrusleeping @diasdavid @lgierth

Comment (diasdavid) [inline contribution-guidelines.md]: it is nice to have always all the labels accessible on every repo, but we don't need to label everything, policing all the issues in smaller projects can consume a lot of time. So yes, but not super strict :)

File: contribution-guidelines.md (hunk 1 above)
Comment (Dignifiedquire) [inline contribution-guidelines.md]: go for it

File: contribution-guidelines.md (hunk 2 above)
Comment (RichardLitt) [inline contribution-guidelines.md]: I would say super strictless enough not to even have it in this guide. Those repos should have their own guides, or just fluid label naming without docs. I think that should work better than specifying it here.

**Comment by dignifiedquire** [comment]
LGTM

**Comment by diasdavid** [comment]
:+1:

**Pull request merged**